Makefile

```diff
@@ -30,7 +30,10 @@ run-test: build-sol
 	poetry run pytest -k $(test)
 
 run-test-mark-log: build-sol
-	poetry run pytest -m $(mark) --log-cli-level=INFO -vvv
+	poetry run pytest -m $(mark) --log-cli-level=INFO -vvv -s
+
+run-test-mark-debug: build-sol
+	poetry run pytest -m $(mark) --log-cli-level=INFO -vvv -s --pdb
```

Comment:
not sure we want to incentivize the use of pdb, visual debugging is the way to go